server/config.coffee

```diff
@@ -33,7 +33,8 @@ config =
             americano.errorHandler
                 dumpExceptions: true
                 showStack: true
-            americano.static path.join __dirname, '/../client/public'
+            americano.static path.join __dirname, '../static'
```

Comment:
This one is a little bit tricky : we can put all frontend assets in the client folder, but they need to be referenced in modules to be built by webpack. It's ok with img, stylesheets, etc. But we also need assets related to the app (here, the favicons and manifests) which are only used in the index page served by the server-part.

So I though it'll be better to:
1. have a `static` folder at root whoch contins all statics stuff related to the whole app (and be copied as-it on build)
2. have another folder in client part containing all stuff linked from modules

I think it's a good and flexible approch, but I'm totally aware to any improvements :smile: